### PR TITLE
Add namespace data to study_es_0

### PR DESCRIPTION
**Pull request by pvannierop**
This PR will: 
- Add numeric and categorical columns for the _Zygosity_ namespace to study_es_0 MAF file.
- Import numerical data in namespace columns as numeric data and expose it as such in the API.
- Add unit and system tests for import of namespace columns.
- Correct the case of imported namespaces to reflect the format shown in the meta file of the MAF file.
- Expand documentation that describes the import of namespace columns.

Note:
- This PR may result in e2e-localdb tests to fail because of incorrect alteration counts.

### CODE REVIEW

File: core/src/main/java/org/mskcc/cbio/maf/MafUtil.java [hunk 1]
```diff
@@ -827,4 +835,40 @@ public static boolean variantContainsAmbiguousTumorSeqAllele(String referenceAll
...
+        if (numberString == null) {
+            return false;
+        }
+        try {
+            new BigDecimal(numberString);
+        } catch (NumberFormatException e) {
+            return false;
+        }
+        return numberString.toUpperCase().contains("E");
+    }
```
[the remark this answers is not in the document]
Comment (pvannierop) [reply core/src/main/java/org/mskcc/cbio/maf/MafUtil.java]: I moved these methods to a different file. Also I updated the implementation somewhat.

**Comment by inodb** [comment]
@pvannierop do you need more review on this or is it good to go?

File: docs/portal.properties-Reference.md [hunk 2]
```diff
@@ -200,6 +200,14 @@ the ability to perform a manual logout and download data access tokens by the us
 skin.hide_logout_button=
 ```
 
+## Namespace columns visible in Mutation Table by default
+Namespace columns are custom columns in the MAF file that can be shown in Mutation Table components. By setting this
+property to _true_ these columns will be visible when the mutation table is created. By default, the namespace column is
+hidden and can be made visible using the column selection menu.
+
+```
+skin.mutation-table.namespace-column.show-by-default=
```
Comment (inodb) [inline docs/portal.properties-Reference.md:209]:
```suggestion
skin.mutation_table.namespace_column.show_by_default=
```
I think all properties have underscores?

File: portal/src/main/webapp/config_service.jsp [hunk 3]
```diff
@@ -130,6 +130,7 @@
             "skin.citation_rule_text",
             "skin.geneset_hierarchy.default_p_value",
             "skin.geneset_hierarchy.default_gsva_score",
+            "skin.mutation-table.namespace-column.show-by-default"
```
Comment (inodb) [inline portal/src/main/webapp/config_service.jsp:133]:
```suggestion
            "skin.mutation_table.namespace_column.show_by_default"
```

File: src/main/resources/portal.properties.EXAMPLE [hunk 4]
```diff
@@ -90,6 +90,9 @@ skin.study_view.link_text=To build your own case set, try out our enhanced Study
 # controls the appearance the logout button on a portal with user authentication
 # skin.hide_logout_button=false
 
+# controls whether namespace columns of the MAF file are immediately visible in mutation tables (default is 'false')
+# skin.mutation-table.namespace-column.show-by-default=true
```
Comment (inodb) [inline src/main/resources/portal.properties.EXAMPLE:94]:
```suggestion
# skin.mutation_table.namespace_column.show_by_default=true
```

**Review by inodb — COMMENTED** [review]
LGTM! One minor thing: i think all properties use underscores, so maybe nice to be consistent

File: docs/portal.properties-Reference.md (hunk 2 above)
Comment (pvannierop) [reply docs/portal.properties-Reference.md]: Updated.

File: portal/src/main/webapp/config_service.jsp (hunk 3 above)
Comment (pvannierop) [reply portal/src/main/webapp/config_service.jsp]: Updated.

File: src/main/resources/portal.properties.EXAMPLE (hunk 4 above)
Comment (pvannierop) [reply src/main/resources/portal.properties.EXAMPLE]: Updated.

**Review by onursumer — APPROVED** [review]

**Pull request merged**